modules/express/test/integration/bitgoExpressV1.ts

```diff
@@ -352,12 +349,12 @@ describe('Bitgo Express', function() {
       })
       .then(function(res) {
         res.should.have.status(200);
-        res.body.size.should.eql(886);
-        res.body.fee.should.eql(17720);
+        res.body.size.should.eql(920);
+        res.body.fee.should.eql(18400);
       });
     });
 
-    it('create and accept a pending approval', function() {
+    xit('create and accept a pending approval', function() {
```

Comment:
why are we canceling all these tests?

Reply:
They come from the v1 integration test suite, which is not run during the normal CI process, since many are broken. These tests are of the broken variety, and thus are disabled off the bat until we can fix.